### ISSUE TITLE
Sonobuoy and Rancher kubernetes engine

### ISSUE THREAD

**Issue by mohjam2004** [vmware-tanzu/sonobuoy#1149]
Hi
I run the e2e test on a rancher kubernetes cluster the result is as follow:
![image](https://user-images.githubusercontent.com/59224403/88803961-7b41ec80-d1c2-11ea-8f24-3768881779ab.png)
most of the tasks are skipped (from 4992 tasks, 4991 tasks are skipped). why?
I have one test cluster with one node as control palne-etcd-worker role.

**Comment by zubron** [comment]
Hi @mohjam2004.

What command did you use to run sonobuoy? It looks like you used the `--mode quick` flag. This mode runs only one test and is intended to be used as an initial check to ensure that the cluster is responding and is reachable before starting a longer test run.

If you omit this flag, Sonobuoy will use the default mode for the e2e plugin which is to run all the conformance tests that are non-disruptive. You can read more about the different e2e modes here: https://sonobuoy.io/docs/v0.18.4/e2eplugin/#built-in-configurations

Even if you use the default mode, you will still see a large number of skipped tests. This is because the image used by Sonobuoy to check conformance contains all the end-to-end tests for Kubernetes. Only a subset of these tests (usually less than 300) are actually required to check conformance: https://sonobuoy.io/docs/v0.18.4/faq/#why-were-so-many-tests-skipped
Most users will not need to run those additional tests unless there is specific behaviour that they wish to check in their cluster (provider specific behaviour, etc.)

I'm going to close this issue, but if you have any further problems, feel free to reopen or open another issue, or come chat to us on [Slack](https://kubernetes.slack.com/messages/sonobuoy).

**Comment by mohjam2004** [comment]
Hi @zubron 
You are right. Now I'm running sonobuoy on default mode to see the results.
That's kind of  you if you send me a invitation link to join rancher work space.
email: mohjam2004@yahoo.com
regards

**Comment by zubron** [comment]
Sonobuoy's slack channel is part of the Kubernetes slack which you can join here: https://slack.k8s.io/. Once signed up, you will be able to join the Sonobuoy channel: https://kubernetes.slack.com/messages/sonobuoy
